Cairo Desktop/CairoDesktop.Common/Localization/DisplayString.cs

```diff
@@ -1,720 +1,857 @@
+                    else if (kvp.Value.StartsWith("sv_"))
+                    {
+                        lang = Language.sv_SE;
+                    }
+                    else if (kvp.Value.StartsWith("zh_"))
+                    {
+                        lang = Language.zh_CN;
+                    }
+                    else if (kvp.Value.StartsWith("cs_"))
+                    {
+                        lang = Language.cs_CZ;
+                    }
+                    else if (kvp.Value.StartsWith("de_"))
+                    {
+                        lang = Language.de_DE;
+                    }
+                    else if (kvp.Value.StartsWith("es_"))
+                    {
+                        lang = Language.es_ES;
+                    }
+                    else if (kvp.Value.StartsWith("nl_"))
+                    {
+                        lang = Language.nl_NL;
+                    }
+                    else if (kvp.Value.StartsWith("ru_"))
+                    {
+                        lang = Language.ru_RU;
+                    }
+                    else if (kvp.Value.StartsWith("pl_"))
+                    {
+                        lang = Language.pl_PL;
+                    }
+                    else if (kvp.Value.StartsWith("tr_"))
+                    {
+                        lang = Language.tr_TR;
+                    }
+                    else if (kvp.Value.StartsWith("ko_"))
+                    {
+                        lang = Language.ko_KR;
+                    }
+                    else if (kvp.Value.StartsWith("hu_"))
+                    {
+                        lang = Language.hu_HU;
+                    }
+                    if (lang != null)
+                    {
+                        for (int i = 0; i < lang.Count; i++)
+                        {
+                            sw.WriteLine(lang.ElementAt(i).Key + "=" + lang.ElementAt(i).Value);
+                        }
+                    }
+                }
+            }
+            //extract languages*/
+
+            if (System.IO.Directory.Exists(System.AppDomain.CurrentDomain.BaseDirectory+ "\\Localization"))
+            {
+                Languages.Clear();
+                string[] lngFiles = System.IO.Directory.GetFiles(System.AppDomain.CurrentDomain.BaseDirectory + "\\Localization", "*.lng");
+                for(int i=0; i< lngFiles.Length; i++) 
+                {
+                    Languages.Add(new KeyValuePair<string, string>(new System.IO.FileInfo(lngFiles[i]).Name.Replace(".lng", ""), new System.IO.FileInfo(lngFiles[i]).Name));
+                }
+            }
+            string lngFile = System.AppDomain.CurrentDomain.BaseDirectory + "\\Localization\\" + Settings.Instance.Language;
+            if (System.IO.File.Exists(System.AppDomain.CurrentDomain.BaseDirectory + "\\Localization\\" + Settings.Instance.Language))
+            {
+                Translation= LoadTranslations(lngFile);
+            }  

```

Comment:
Should be able to pass `lngFile` to `File.Exists()` instead of duplicating the string.

Reply:
Changed, I also have no idea why I did it in such a weird way. I also made so that Localization folder is a string and lngFile is based on that.